assets/js/archive-status.js

```diff
@@ -0,0 +1,63 @@
+// Load data attributes
```

Comment:
AFAIK, this is not necessary, since our `block extrabody` template tag is placed right before `</body>`
https://github.com/st-tu-dresden/inloop/blob/49ed599010c990e137ded09ca1c3acb0fb8f56c6/inloop/templates/base.html#L85
But to make it proof for exotic html processing schemes, i will add this in.